### DISCUSSION TITLE
scrollable vpack

### DISCUSSION THREAD

**Discussion by lancecarlson** [fltk-rs/fltk-rs#1065] [Q&A] [answered]
I need to make an infinitely scrollable "table" (up and down), where the data inside the cells will be updating in real time. Is it recommended that I use a table of some sort or a vpack and hpack combination for performance? I want to constrain the heights of each row but allow the widths to be adjustable (on a column level). I also want to style the columns, and make certain cells clickable etc. 

Side note:
I'm about to dig into the spreadsheet example because the table example is too redumentary for my needs and the smart table doesn't do everything I need either. I'm also worried about performance.

Thanks for your hard work on this library!

**Comment by MoAlyousef** [comment, marked as answer]
Thanks for your words. It seems the FLTK docs don't recommend tables as large widget containers, which is what you need since you would like clickable cells. So I would suggest going with packs, they're also more flexible since you can easily add packs while for tables you need to reset the number of columns and rows then redraw.
